vulnerability-disclosure-policy.md

```diff
@@ -67,3 +67,27 @@ Reports should include:
 Please keep your vulnerability reports current by sending us any new information as it becomes available.
 
 We may share your vulnerability reports with [US-CERT](https://www.us-cert.gov/ais), as well as any affected vendors or open source projects.
+
+### Public Disclosure
+
+This public disclosure policy applies to all submissions under the TTS Vulnerability Disclosure Policy, including Duplicates, Out of Scope, and Not Applicable submissions.
+
+#### Nondisclosure
+
+For 90 days, TTS's default submission disclosure state is nondisclosure for 90 days after submission. Within this 90 day time period, no submissions may be publicly disclosed in full or part without explicit permission from TTS.
+
+During nondisclosure, TTS may patch or update our open source code in response to a submission. This is not public disclosure.
```

Comment:
It very much is public disclosure, which is why open source projects typically distribute patches and advisories on the same day.

Reply:
How can we talk about disclosure of the communication/report on H1 with the patch? I didn't know how to describe these separately. Would welcome edits.